source/resiliency.txt

```diff
@@ -12,12 +12,140 @@ Application and Database Resiliency
+at the same time. However, if there is a large range in size between your 
+minimum and maximum pool size, additional connections are opened more frequestly 
+during application runtime. 
+
+This process of incrementally increasing your connection pool size during 
+application runtime distributes the total workload of connecting clients from 
+your application to |service| over a longer period of time, which often makes it 
+manageable for a given use case, but it is important to note that the associated 
+increase in network load occurs during application runtime, which has 
+the potential to impact perceived database - and by extension - application 
+performance for end-users.
+
+Your application's architecture is central to this consideration. If, for example, 
+you deploy your application as microservices in an elastic environment, consider 
+which services should call |service| directly as a means of controlling the 
+dynamic expansion and contraction of your connection pool. 
+
+Optimizing Request Configurations for Distinct Query Types
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Query Timeout
+`````````````
+Almost invariably, workload-specific queries from your application will vary in 
+terms of the amount of time they take to execute in |service| and in terms of 
+the amount of time your application can wait for a response. 
+
+Consider defining query classes that handle categories or buckets of similar 
+request requirements. For example, you can define a query category with a fast 
+timeout for end-user driven requests, a middle tier timeout bucket for general 
+purpose requests, and a long-running query class for things like analytics 
+queries that require the most time to execute in |service|. 
+
+`Query timeout <https://www.mongodb.com/docs/manual/tutorial/query-documents/specify-query-timeout/>`__ behavior can be set globally in |service| and it can be defined 
+at the query level. 
+
+Retryable Database Reads and Writes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service| supports `retryable read <https://www.mongodb.com/docs/manual/core/retryable-reads/>`__ and `retryable write <https://www.mongodb.com/docs/manual/core/retryable-writes/>`__ operations. When enabled, 
+read and write operations are retried once as a safeguard against intermittent 
+network outages. 
+
+Configure Read and Write Concerns 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|service| {+clusters+} eventually replicate all data across all nodes. However, 
+you can configure the number of nodes across which data must be repicated before 
+a read or write operation is reported to have been successful. `Read concerns <https://www.mongodb.com/docs/manual/reference/read-concern/>`__ and 
+`write concerns <https://www.mongodb.com/docs/manual/reference/write-concern/>`__ can be defined globally in |service| and they can be defined at the 
+client level in the connection string. 
 
 Examples
 --------
```

Comment:
If they can't set these things in the CLI or TF, let's remove the example section from this page.

Reply:
I went ahead and removed for now, as I can't yet access dev clusters with working API keys, and I think a description and link out to the existing docs works here. Happy to adjust as needed. Thanks